### PR TITLE
Added HoverEffect

### PR DESCRIPTION
**Pull request by RakeshCharpe**
# Description

I Added Hover effect in navbar which enhance our overview of the website.

Fixes issue #50 

## Type of change

- [x ] Bug fix (non-breaking change which fixes an issue)
- [x ] New feature (non-breaking change which adds functionality)

# Explain the Testing instructions

**Test Configuration**:
* Operating system:
* Version:
* Text-editors used:
 
# Checklist:

- [x ] My code follows the style guidelines of this project
- [x ] I have performed a self-review of my own code
- [x ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x ] My changes generate no new warnings
- [x ] I have added tests that prove my fix is effective or that my feature works
- [x ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

# ATTACH SCREEN-SHOTS / DEPLOYMENT LINK 
Screenshot (69)

### CODE REVIEW

**Review by Ayan-thecodeking — CHANGES_REQUESTED** [review]
hey @RakeshCharpe the branch has conflicts plz resolve this conflicts and then make the PR

File: index.html [hunk 1]
```diff
@@ -18,10 +18,10 @@
     <nav class="navigation-bar">
         <div class="logo">Coders Evoke</div>
         <ul class="nav-links">
-            <li><a href="#home">Home</a></li>
-            <li><a href="">About</a></li>
-            <li><a href="">Project</a></li>
-            <li><a href="">Contact</a></li>
+            <li><a class="bar"href="#home">Home</a></li>
```
Comment (RakeshCharpe) [inline index.html:21]: Added  class: bar  into < li >tag.

File: assets/css/style.css [hunk 2]
```diff
@@ -69,7 +71,30 @@ div.logo {
   background-color: #fff;
   transition: all 0.5s ease-in-out;
 }
-
```
Comment (RakeshCharpe) [inline assets/css/style.css:72]: Added hover effect functionality in css files from this lines

**Review by RakeshCharpe — COMMENTED** [review]
I added the comments which resolve conflicts

**Comment by RakeshCharpe** [comment]
@Ayan-thecodeking I resolve conflict merge into main.

**Comment by RakeshCharpe** [comment]
@neelshah2409 I already worked on this issue and my PR is merged as well

**Comment by Ayan-thecodeking** [comment]
> @neelshah2409 I already worked on this issue and my PR is merged as well

Yup ...we are adding labels

**Pull request merged**